js/app.js

```diff
@@ -0,0 +1,263 @@
+'use strict'

```

Comment:
Using strict mode in my code helps me write better code. In browsers supporting strict mode, it eliminates some of the silent errors to throw errors. It also prevents duplicate keys in an object, use of undeclared variables, duplicate arguments in a function thereby making sure that I write good code.